### PR TITLE
Tell about SYMFONY__TEMPLATING__HELPER__CODE__FILE_LINK_FORMAT

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -261,8 +261,10 @@ you use PHPstorm on the Mac OS platform, you will do something like:
 
 Of course, since every developer uses a different IDE, it's better to set
 this on a system level. This can be done by setting the ``xdebug.file_link_format``
-in the ``php.ini`` configuration to the url string. If this configuration
-value is set, then the ``ide`` option will be ignored.
+in the ``php.ini`` configuration to the url string.
+An other way if you don't use Xdebug is to set this url string in the
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: `An other` -> `Another`

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -261,8 +261,10 @@ you use PHPstorm on the Mac OS platform, you will do something like:
...
+``SYMFONY__TEMPLATING__HELPER__CODE__FILE_LINK_FORMAT`` environement variable.
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: `environement` -> `environment`

**Comment by javiereguiluz** [comment]
:+1:

A minor comment to our doc managers: in https://github.com/symfony/symfony/pull/15549 there is a pending merge to fix some issue related to the `framework.ide` option. But this PR can be safely merged without waiting for getting that bugfix merged. Thanks!

File: reference/configuration/framework.rst [hunk 3]
```diff
@@ -261,8 +261,10 @@ you use PHPstorm on the Mac OS platform, you will do something like:
 
 Of course, since every developer uses a different IDE, it's better to set
 this on a system level. This can be done by setting the ``xdebug.file_link_format``
-in the ``php.ini`` configuration to the url string. If this configuration
-value is set, then the ``ide`` option will be ignored.
+in the ``php.ini`` configuration to the url string.
+Another way if you don't use Xdebug is to set this url string in the
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: Should this be a new paragraph?

File: reference/configuration/framework.rst [hunk 4]
```diff
@@ -261,8 +261,10 @@ you use PHPstorm on the Mac OS platform, you will do something like:
 
 Of course, since every developer uses a different IDE, it's better to set
 this on a system level. This can be done by setting the ``xdebug.file_link_format``
-in the ``php.ini`` configuration to the url string. If this configuration
-value is set, then the ``ide`` option will be ignored.
+in the ``php.ini`` configuration to the url string.
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: URL (same below)

File: reference/configuration/framework.rst [hunk 5]
```diff
@@ -261,8 +261,10 @@ you use PHPstorm on the Mac OS platform, you will do something like:
...
+Another way if you don't use Xdebug is to set this url string in the
+``SYMFONY__TEMPLATING__HELPER__CODE__FILE_LINK_FORMAT`` environment variable.
+If any of these configurations values are set, then the ``ide`` option will be ignored.
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: I think "then" can be removed here.

File: reference/configuration/framework.rst (hunk 3 above)
Comment (xabbuh) [inline reference/configuration/framework.rst]: Should "if you don't use Xdebug" be enclosed by commas?

**Comment by xabbuh** [comment]
@nicolas-grekas Can you finish here?

**Comment by nicolas-grekas** [comment]
updated

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @nicolas-grekas.

**Pull request merged**